internal/agent/delta/store.go

```diff
@@ -87,6 +89,58 @@ type Store struct {
 	CacheDir string
 	// NextIDMap stores the information about the available plugins
 	NextIDMap pluginIDMap
+	// stores time of last success submission of inventory to backend
+	lastSuccessSubmission time.Time
+}
+
+func (d *Store) LastSuccessSubmission() (time.Time, error) {
```

Comment:
Could be decoupled from Store? sounds like LastSuccessSubmission has its own responsibility -> persist a time value 🤔

Reply:
sure thing